index.php

```diff
@@ -1,140 +1,91 @@
 <?php
-    session_start();
-    if(session_is_registered(username)) {
-        header("location:admin.php");
-    }
+    require_once 'core/init.php';      
+    if(!User::isLoggedIn())       
```

Comment:
The index page is supposed to be a logged-out experience, i.e. like the homepage of a site where everyone can come in and look at top students and other public data in the college (e.g. student list branch wise).

Admin.php is supposed to be a logged-in experience where an admin can add/manage records.

So I believe checking for login on index isn't a good idea, rather we should check for admin login and redirect to admin.php if true.